interfaces/builtin/basedeclaration.go

```diff
@@ -195,9 +195,28 @@ slots:
       slot-snap-type:
         - app
         - gadget
+    allow-connection:
+      -
+        plug-attributes:
+          content: $SLOT(content)
```

Comment:
Is this meaning to say that the connection should be allowed if the content attribute of the plug matches the content attribute of the slot? It seems so, but want to make doubly sure.